server/controllers/admin/userManagementController.ts

```diff
@@ -22,6 +24,21 @@ export default class UserController {
     }
   }
 
+  update(): TypedRequestHandler<Request, Response> {
+    return async (req: Request, res: Response) => {
+      const user = await this.userService.getUserById(req.user.token, req.params.id)
+
+      await this.userService.updateUser(req.user.token, {
+        ...user,
+        roles: [...flattenCheckboxInput(req.body.roles), ...flattenCheckboxInput(req.body.allocationPreferences)],
+        qualifications: flattenCheckboxInput(req.body.qualifications),
+      })
+
+      req.flash('success', 'User updated')
+      res.redirect(`${paths.admin.userManagement.show({ id: user.id })}?success=true`)
```

Comment:
What does the `success=true` arg do here?

Reply:
I left this in in error. 
In CAS3 they do this when the page that the user is directed to on success is the same as the one if there is a validation error for auditing purposes. I thought we would need it here but as there isn't any validation we didn't end up needing it